composer.json

```diff
@@ -12,7 +12,8 @@
     "license": "MIT",
     "require": {
         "doctrine/orm": ">=2.3",
-        "creof/geo-parser": ">=1.0"
+        "creof/geo-parser": ">=1.0",
```

Comment:
You should probably use ~1.0 which limit the version >=1.0 && < 2.0. So if you release a `2.0` that have BC issue, this is not going to have side effect on this library.